src/explore-education-statistics-common/src/components/ReorderableItem.tsx

```diff
@@ -0,0 +1,100 @@
+import DragIcon from '@common/components/DragIcon';
+import styles from '@common/components/ReorderableItem.module.scss';
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import { ArrowLeft, ArrowRight } from '@common/components/ArrowIcons';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import { mergeRefs } from '@common/utils/mergeRefs';
+import classNames from 'classnames';
+import React, { ReactNode, useEffect, useRef } from 'react';
+import { DraggableProvided, DraggableStateSnapshot } from '@hello-pangea/dnd';
+
+export interface ReorderableListItem {
+  id: string;
+  label: ReactNode | string;
+}
+
+interface Props {
+  draggableProvided: DraggableProvided;
+  draggableSnapshot: DraggableStateSnapshot;
+  focusItem: boolean;
+  index: number;
+  isLastItem: boolean;
+  item: ReorderableListItem;
+  onMoveItem: ({
+    prevIndex,
+    nextIndex,
+  }: {
+    prevIndex: number;
+    nextIndex: number;
+  }) => void;
+}
+
+export default function ReorderableItem({
+  draggableProvided,
+  draggableSnapshot,
+  focusItem = false,
+  index,
+  isLastItem,
+  item,
+  onMoveItem,
+}: Props) {
+  const itemRef = useRef<HTMLLIElement>(null);
+  useEffect(() => {
+    if (focusItem) {
+      itemRef.current?.focus();
+    }
+  });
+
+  return (
+    <div className={styles.container}>
+      <div
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...draggableProvided.draggableProps}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...draggableProvided.dragHandleProps}
+        className={classNames('govuk-!-margin-bottom-0', styles.draggable, {
+          [styles.isDragging]: draggableSnapshot.isDragging,
+          [styles.isDraggedOutside]:
+            draggableSnapshot.isDragging && !draggableSnapshot.draggingOver,
+        })}
+        ref={mergeRefs(draggableProvided.innerRef, itemRef)}
+        style={draggableProvided.draggableProps.style}
+        data-testid="reorderable-item"
+      >
+        <div className={styles.itemLabel}>
+          <DragIcon className={styles.dragIcon} />
+          <span>{item.label}</span>
+        </div>
+      </div>
+      {!draggableSnapshot.isDragging && (
```

Comment:
![image](https://github.com/user-attachments/assets/a341bef0-f6d8-4acb-b037-a56e344962f0)

In the image above, you'll notice the UP button from footnote 3 has moved up into the empty drop area - I wonder if we might hide move buttons for all items during drag action

Reply:
Good spot, that's sorted now - the buttons are hidden when something is being dragged.